src/main.jl

```diff
@@ -59,21 +28,23 @@ function help_message()
 end
 
 function start_bot(;
-    commands = active_commands,
-    handlers = handlers_list,
-    run_duration = Days(365),  # run for a very long time by default
+    commands=active_commands,
+    handlers=handlers_list,
+    run_duration=Days(365),  # run for a very long time by default
 )
     @info "Starting bot... command prefix = $COMMAND_PREFIX"
-    global client = Client(ENV["HOJBOT_DISCORD_TOKEN"];
-        presence = (game = (name = "HoJ", type = AT_GAME),),
-        prefix = COMMAND_PREFIX)
+    global client = Client(
+        ENV["HOJBOT_DISCORD_TOKEN"];
+        presence=(game=(name="HoJ", type=AT_GAME),),
+        prefix=COMMAND_PREFIX,
+    )
     init_handlers!(client, handlers)
     init_commands!(client, commands)
     # add_help!(client; help = help_message())
     warm_up()
     open(client)
     auto_shutdown(client, run_duration, "SHUTDOWN")
-    wait(client)
+    return wait(client)
```

Comment:
I wonder what `wait` returns here. It's probably not that useful anyways since `start_bot` is called from the run.sh script.

Actually, what would be more useful is to wrap the body of this function (`start_bot`) with a try-catch block. If any exception is thrown then you can log the error and call `exit` with a non-zero status.  Then, the run.sh script could have a chance to do something different, if we wish, knowing that something went wrong.  It's kinda minor though; and if you want to do it, then I would suggest submitting that patch as a separate PR.